client/rhel/mgr-daemon/mgr-daemon.spec

```diff
@@ -28,21 +35,25 @@ Version:        4.0.1
 Release:        1%{?dist}
 Summary:        Spacewalk query daemon
 License:        GPL-2.0-only
+%if %{_vendor} == "debbuild"
+Group:          utils
+Packager:       Spacewalk Project <spacewalk-devel@redhat.com>
+%else
 Group:          System Environment/Base
+%endif
 Source0:        spacewalksd-%{version}.tar.gz
 Source1:        %{name}-rpmlintrc
 URL:            https://fedorahosted.org/spacewalk
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
-BuildRequires:  gettext
+%if %{_vendor} != "debbuild"
+
 # 5.0.37.2 was last version+1 of spacewalksd before renaming to mgr-daemon
```

Comment:
Uhh, what?

Reply:
The comment refer to the Provides/Obsoletes statements below.
We renamed that package and by this we changed the version to a lower version then the old
package had.

mgr-daemon 4.0.1 obsolete rhnsd   < 5.0.37.2